src/main/java/uk/gov/hmcts/reform/blobrouter/exceptions/DocSignatureFailureException.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.blobrouter.exceptions;
+
+/**
+ * Exception representing a failure because of an invalid signature.
+ */
+public class DocSignatureFailureException extends RuntimeException {
```

Comment:
What's the difference between `DocSignatureFailureException` and `SignatureValidationException`?

Reply:
`DocSignatureFailureException` - for invalid signature and the other is used for generic failure when validating the signature. Added a comment in `SignatureValidationException` class